testing/scenario_app/lib/src/platform_view.dart

```diff
@@ -437,13 +438,7 @@ class MultiPlatformViewBackgroundForegroundScenario extends Scenario
     _nextFrame();
   }
 
-  bool _firstFrameBegan = false;
-
   void _firstFrame() {
-    if (!_firstFrameBegan) {
-      channelBuffers.setListener('flutter/lifecycle', _onPlatformMessage);
-    }
-    _firstFrameBegan = true;
```

Comment:
is this change also necessary to fix the issue? will moving this call from `firstFrame` to the `init` cause any problem?

Reply:
This change is basically just undoing my first attempt at a fix (https://github.com/flutter/engine/pull/48096), which was still flakey. So no, it shouldn't cause any problems as this was as the code originally was